app/src/main/res/values/strings.xml

```diff
@@ -135,6 +135,10 @@
     <string name="client_xsdl">XSDL</string>
 
     <!-- Alerts -->
+    <string name="button_yes">Yes!</string>
+    <string name="button_negative_1">Not really</string>
+    <string name="button_negative_2">No, thanks</string>
```

Comment:
Could you change the names of these buttons to be more descriptive than numbers?